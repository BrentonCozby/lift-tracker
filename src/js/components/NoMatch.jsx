import React from 'react'

const NoMatch = () => (
    <div className="NoMatch">
        <h1>Page not found</h1>
    </div>
)

export default NoMatch
