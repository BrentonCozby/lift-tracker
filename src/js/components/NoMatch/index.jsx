import React from 'react'

export default () => (
    <div className="NoMatch">
        <h1>Page not found</h1>
    </div>
)
