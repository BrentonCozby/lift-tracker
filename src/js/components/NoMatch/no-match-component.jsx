import React from 'react'

export default function NoMatch() {
    return (
        <div className="NoMatch">
            <h1>Page not found</h1>
        </div>
    )
}
