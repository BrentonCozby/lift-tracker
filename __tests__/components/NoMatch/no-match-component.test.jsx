import React from 'react'
import renderer from 'react-test-renderer'
import NoMatch from '../../../src/js/components/NoMatch/no-match-component.jsx'

function setup() {
    const props = {

    }

    const componentJSON = renderer.create(<NoMatch {...props} />).toJSON()

    return {
        props,
        componentJSON
    }
}

describe('NoMatch', () => {
    it('renders self and subcomponents', () => {
        const { componentJSON } = setup()

        expect(componentJSON).toMatchSnapshot()
    })
})