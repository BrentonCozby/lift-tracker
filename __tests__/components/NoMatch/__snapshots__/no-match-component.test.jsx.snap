// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch renders self and subcomponents 1`] = `
<div
  className="NoMatch"
>
  <h1>
    Page not found
  </h1>
</div>
`;
